product_docs/docs/migration_toolkit/55/02_supported_operating_systems_and_database_versions.mdx

```diff
@@ -24,12 +24,12 @@ The following database product versions can be used with Migration Toolkit:
 -   MySQL 8.0
 -   Sybase Adaptive Server Enterprise 15.7
 
-Please contact your EnterpriseDB Account Manager or [sales@enterprisedb.com](mailto:sales@enterprisedb.com) if you require support for other database products.
+Contact your EnterpriseDB Account Manager or [sales@enterprisedb.com](mailto:sales@enterprisedb.com) if you require support for other database products.
 
 !!! Note
-    The Migration Toolkit has not been tested and does not officially support use with Oracle Real Application Clusters (RAC) and Exadata. However, Migration Toolkit may work when it is connected to a single persistent node. For more information, contact your EDB representative.
+    The Migration Toolkit wasn't tested with and doesn't officially support use with Oracle Real Application Clusters (RAC) and Exadata. However, Migration Toolkit might work when it is connected to a single persistent node. For more information, contact your EDB representative.
```

Comment:
Should we replace this with "isn't"?

"Wasn't" might imply that it has now been tested.

Reply:
good edit!